app/controllers/userAuth.js

```diff
@@ -0,0 +1,78 @@
+const jwt = require('jsonwebtoken');
```

Comment:
This file is the same as app/controllers/users.js, copy your update to app/controllers/users.js.